### ISSUE TITLE
Solver should be aware of (solve) ALL biomes 

### ISSUE THREAD

**Issue by bpbond** [JGCRI/hector#19]
Major   Solver should be aware of (solve) ALL biomes in simpleNbox  N   simpleNbox.cpp      Ben     Open

**Comment by kdorheim** [comment]
Should follow up to see if this issue is still relevant, I suspect that @ashiklom might have already implemented this.

**Comment by ashiklom** [comment]
Yes, still relevant, I think. The current multi-biome approach is a hack all the biomes are aggregated, then the differential equations are solved, and then the resulting global fluxes are disaggregated (IIRC according to global NPP + RH share? Or maybe biomass?). It's a reasonable hack that probably comes close to the more formal approach (where the biome-specific differential equations are solved), but a hack nonetheless.

**Comment by bpbond** [comment]
More generally, the whole solver interface with simpleNbox needs to be tackled at some point. Right now the ocean model is linked through there...it's kludgey and ugly.
